scylla-cdc/src/stream_reader.rs

```diff
@@ -7,10 +7,11 @@ use futures::StreamExt;
 use itertools::{repeat_n, Itertools};
 use scylla::frame::response::result::CqlValue;
 use scylla::Session;
+use scylla_cdc_utils::ToTimestamp;
```

Comment:
We are using `ToTimestamp` in just two places - one of them is in the non-test code due to which we need to specify `scylla-cdc-utils` in dependencies, not dev-dependencies. Can we get rid of it? It doesn't save much typing and I don't think it is worth keeping in these circumstances.

Reply:
OK, so we _did_ use it in multiple places, I'm not sure why I thought otherwise this morning... Nevertheless, I think the switch to `now()` from utils is fine.